terraform/aks/workspace_variables/app_config.yml

```
@@ -0,0 +1,12 @@
+default: &default
+  RAILS_SERVE_STATIC_FILES: true
+
+review_aks:
+  <<: *default
+  RAILS_ENV: review_aks
+  RACK_ENV: review_aks
+
+dv_review_aks:
+  <<: *default
+  RAILS_ENV: review_aks
+  RACK_ENV: review_aks
```

```suggestion
default: &default
  RAILS_SERVE_STATIC_FILES: true
  RAILS_ENV: review_aks
  RACK_ENV: review_aks
  
review_aks:
  <<: *default

dv_review_aks:
  <<: *default
```

Something like this if `dv_review_aks` is meant to share the same env

What is the difference by the way?
`dv_review_aks` vs `review_aks` 

`dv` is `dev` right?

---

Nice. Yea dv is dev, mainly for the platform team